grants/speculative/Polkadot_Central_Europe_Community_Development.md

```diff
@@ -0,0 +1,90 @@
+## Grant Application
+
+This application is (select one):
+- [x] Speculative
+- [ ] an RFP response
+
+
+###
+
+This application is (select one):
+- [ ] Public (fully)
+- [x] Public with private finances
+
+
+### Abstract
+
+[Provide a brief description of your project here (summarise key points).
+
+If your application is part of a programme, please mention which in the first line of the abstract.
+
+The details should be in the application 'document' that you add to this repo via a pull request.]
+
+
+## Checklist
+- [x] The [grants](https://github.com/w3f/Web3-collaboration/blob/master/grants/grants.md) document has been read and understood.
+- [x] The [Google Form](https://docs.google.com/forms/d/e/1FAIpQLSfMfjiRmDQDRk-4OhNASM6BAKii7rz_B1jWtbCPkUh6N7M2ww/viewform) will be completed accurately. Note that the Google Form requires the pull request URL.
+- [x] Abstract (above) is succinct and complete.
+- [x] The application is being included into the correct directory: either '[targeted](https://github.com/w3f/Web3-collaboration/tree/master/grants/targeted)' or '[speculative](https://github.com/w3f/Web3-collaboration/tree/master/grants/speculative)'.
+- [x] The application includes a project description.
+- [x] The application includes all names of team members.
+- [x] The application includes a description of the team's experience.
+- [x] The application includes all necessary GitHub and LinkedIn links.
+- [x] The application specifies the development language and the reason for choosing it.
+- [x] The "Development Roadmap" section in the application has a timeline of development.
+- [x] The "Development Roadmap" section in the application has an estimate of funds required.
+- [x] The "Development Roadmap" section gives an indication of the team's long term plans.
+
+
+
+
+
+-------------------------------------------------------------------------------------------------------------------------------
+
+# Polkadot Central Europe Community Building
+
+## Project Description
+We want to raise engagement in Central Europe blockchain community about Polkadot and Web3 related technologies (Substrate, libp2p,etc.) by organising meetups, workshops and hackathons in our blockchain cowork space Progressbar in Bratislava and other cities nearby - Vienna, Prague. Events will be streamed and followed by videos and blogs afterwards.
+
+## Team members
+* Ondrej Sarnecky
+* Anett Rolikova
+* Jakub Gregus
+
+## Team Website
+* https://cowork.progressbar.sk/
+
+## Legal Structure
+Non-profit
+
+## Team's experience
+Ondrej Sarnecky acts as a consultant of blockchain technology. Nowadays he is actively involved in client's projects ranging from gambling to real estate, where his main duties are the token sale preparation, designing smart contracts and the blockchain technology implementation in his client's bussinesses. He is a prolific blog writer concerned with cryptocurrencies and blockchain technology implementation.
+
```

Comment:
Anette Rolikova wanted to help grow blockchain community in cowork space. She is helping with creating agenda for events happening in blockchain focused cowork space Progressbar. She volunteered on blockchain conferences like Aracon and Goerlicon. She will help host these events and build community around web3 devs.